app/models/user.rb

```diff
@@ -71,7 +71,6 @@ class User < ApplicationRecord
   validate :password_complexity
 
   default_scope -> { kept }
-  scope :alphabetized, -> { order(discarded_at: :desc, name: :asc) }
```

Comment:
This wasn't being used anywhere, so I opted to remove it and look to order user select fields on the instance vars defined in relevant controller files.